doc/source/conf.py

```diff
@@ -41,6 +42,15 @@
     '.md': 'recommonmark.parser.CommonMarkParser',
 }
 
+# Generate JSON version of the helm-chart
+import yaml, json, sys
```

Comment:
Small style suggestion: Since `yaml` is coming from PyYAML and not the standard lib, it's probably better to import on its own line.

Reply:
+1, let's import things separately all the time :)

Reply:
oops - that was copypasta from a stackoverflow :-)